owners.php

```diff
@@ -50,4 +50,4 @@ function hideDogInfo(element) {
     var dogInfo = element.nextElementSibling;
     dogInfo.style.display = "none";
   }
-</script>
\ No newline at end of file
+</script>
```

Comment:
Ten výpis všech majitelů by šel udělat lépe tak, že si předem připravíš (vrátíš) co je třeba a pak to v templatu jen chytře použiješ.

Místo var v Javascriptu používat **const/let**.